src/frontend/sider/Fagsak/Behandling/Sider/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -51,7 +51,7 @@ const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ åpenBehand
         erUlagretNyRefusjonEøsPeriode,
     } = useVedtakSteg();
 
-    const { behandlingErMigreringMedAvvikUtenforBeløpsgrenser } = useSimulering();
+    const { behandlingErMigreringMedAvvikUtenforBeløpsgrenser } = useSimuleringContext();
```

Comment:
Det irriterer meg veldig at vi trenger å bruke simulering context i denne filen. Dette er en komponent som vises på 'Vedtak'-siden og som gjør at vi trenger å wrappe hele vedtak-siden i SimuleringProvider. Siden det er ulike instanser av provider på simulering-siden og vedtak-siden gjør det at man feks må kalle backend endepunkt for similulering to ganger etter hverandre....